Tutorial/stopwatch.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+// Functionality to measure the duration (millisecond) of event takes place
+namespace yugen {
+namespace stopwatch {
+
+class StopWatch {
+ private:
+  // end_time_ is set during initialize
+  long end_time_;
+
+  // elapsed_ in milliseconds will increase at every event.
+  long elapsed_;
+
+  // triggered_ is set to true if StopWatch hits end_time value
+  bool triggered_;
+
+ public:
+  StopWatch() = default;
+
+  // sets end_time value
+  void Initialize(long end_time);
+
+  // start stop watch
+  void Start();
+
+  // stop stop watch
+  void Stop();
+
+  // get elapsed_ value
+  long get_value() const;
```

Comment:
Name of function should follow name of property. In this case, should be get_elapsed

```suggestion
  long get_elapsed() const;
```